### PR TITLE
[docs] Add note about DuckDB in lakeFS WebUI to DuckDB integrations page

### LINKED ISSUE #6196: [docs] Link to docs about duckDB querying in UI
On https://docs.lakefs.io/integrations/duckdb.html it would be useful to link to the doc about how you can use duckDB from within the lakeFS UI

### PR DESCRIPTION
**Pull request by rmoff**
### Linked Issue

Closes #6196

### CODE REVIEW

File: docs/integrations/duckdb.md [hunk 1]
```diff
@@ -72,3 +73,19 @@ USING SAMPLE reservoir(50000 ROWS) REPEATABLE (100);
 
 COPY sampled_population TO 's3://example-repo/main/data/population/sample.parquet'; -- actual write happens here
 ```
+
+## Using DuckDB in the lakeFS Web Interface
```
Comment (nopcoder) [inline docs/integrations/duckdb.md:77]: Usually in the docs we reference it as UI or web UI I think

**Review by nopcoder — APPROVED** [review]
LGTM - just small comment about a title

File: docs/integrations/duckdb.md (hunk 1 above)
Comment (rmoff) [reply docs/integrations/duckdb.md]: thanks, fixed.

File: docs/integrations/duckdb.md [hunk 2]
```diff
@@ -72,3 +73,19 @@ USING SAMPLE reservoir(50000 ROWS) REPEATABLE (100);
...
+
+The lakeFS web UI includes DuckDB in the Object viewer page. 
+
+![DuckDB query editor on the lakeFS objects page](/assets/img/duckdb.png)
+
+Using this you can query objects in lakeFS directly using a `lakefs` path: 
+
+    lakefs://<repository>/<branch>/object/path/foo.parquet
+
+The DuckDB query editor is provided by [DuckDB WASM](https://github.com/duckdb/duckdb-wasm). It will be used to render and provide querying capabilities for any objects of the following types:
```
Comment (arielshaqed) [inline docs/integrations/duckdb.md:87]:
```suggestion
The DuckDB query editor is provided by [DuckDB WASM](https://github.com/duckdb/duckdb-wasm). It renders and provides querying capabilities for any objects of the following types:
```

**Review by arielshaqed — APPROVED** [review]
Cool neat yay!

**Pull request merged**